jobs/e2e-test-cypress.groovy

```diff
@@ -68,8 +71,8 @@ to spin up.""",
 )
 .addStringParam(
    "TEST_RETRIES",
-   """How many retry attempts to use. By default is 3.""",
-   "3"
+   """How many retry attempts to use. By default is 4.""",
+   "4"
```

Comment:
**note:** I'm increasing the # of retries to see if we get more consistent results for a couple of flaky tests.